### ISSUE TITLE
workflow for vignettes

### ISSUE THREAD

**Issue by f-hafner** [carpentries-incubator/lesson-R-packaging#96]
Not sure how relevant this is in practice, but how much detail do you want to cover the workflow for making vignettes? (what is the experience from previous workshops)? This is related to the discussion on reloading the package in https://github.com/carpentries-incubator/lesson-R-packaging/pull/92, and if the conclusion there is to use `install()`, then this issue can be closed.

I have been trying to incorporate some of the suggestions for writing vignettes from the [book](https://r-pkgs.org/vignettes.html#sec-vignettes-how-built). I understand them as follows:
1. write vignette, discover that you want to change `myfunction` in package. 
1. change `myfunction`.
2. run `devtools::load_all()` *in the console* (and not in an Rmd chunk)
3. update the vignette Rmd 
4. iterate until happy
5. make a "disposable preview" of the html with either the `knit` button, or `devtools::build_rmd("vignettes/my-vignette.Rmd")`. I think the difference is that `devtools::build_rmd()` automatically considers the changes in the package made, while using the `knit` button will use the currently installed local version of the package. and if we used `load_all()` in step 2 above, the resulting html file will not reflect these changes.

So, in short, if we recommend using `load_all()` for development in previous episodes, we may have to discuss the above in the episode on vignettes.

**Comment by bvreede** [comment]
I addressed this with #113  — forgot to tag the issue though! Sorry :)
